mgmtfn/k8splugin/driver.go

```diff
@@ -448,7 +476,7 @@ func addPod(r *http.Request) (interface{}, error) {
 	}
 
 	// Set default gateway
-	err = setDefGw(pid, gw, gwIntf)
+	err = setDefGw(pid, gw, ep.IPv6Gateway, gwIntf)
```

Comment:
include ipv6 address/gateway in CNI response  https://github.com/contiv/netplugin/tree/master/mgmtfn/k8splugin/contivk8s